### ISSUE TITLE
Docs need update

### ISSUE THREAD

**Issue by PhilD13** [ForumPostAssistant/FPA#14]
Docs need updating to reflect current version

**Comment by mandville** [comment]
I assumed that with most cases, readme's arent. also the docs on the J.site have had the versions removed. Perhaps just reword it to "all stable releases" 
better test it with 3 at some point

**Comment by PhilD13** [comment]
Thanks for the test with 3 reminder.

**Comment by PhilD13** [comment]
Quick check of Joomla! 3.0.0_Alpha1 Alpha [ Ember ] 9-July-2012 on a production server . FPA works but not without some errors as outlined below.

Environment Platform:
Linux 
Kernel Version:
2.6.32.21-grsec
PHP API
litespeed
MySQL Connection Type
mysqli
PHP Version
5.3.6
MySQL Version
5.0.87-community
Zend Engine
2.3.0

 Top info box Errors:

PHP Supports J!3.0
Unknown

MySQL Supports J!3.0
Unknown

Application Instance :: Discovery info box errors:
Config Version
2.5 and above
CMS Mis-Match
